dali/python/bundle-wheel.sh

```diff
@@ -88,37 +89,21 @@ make_wheel_record() {
 }
 
 DEPS_LIST=(
-    "/usr/local/lib64/libjpeg.so.62"
-    "/usr/local/lib/libavformat.so.58"
-    "/usr/local/lib/libavcodec.so.58"
-    "/usr/local/lib/libavfilter.so.7"
-    "/usr/local/lib/libavutil.so.56"
-    "/usr/local/lib/libtiff.so.5"
-    "/usr/local/lib/libsndfile.so.1"
-    "/usr/local/lib/libFLAC.so.8"
-    "/usr/local/lib/libogg.so.0"
-    "/usr/local/lib/libvorbis.so.0"
-    "/usr/local/lib/libvorbisenc.so.2"
-    "/usr/local/lib/libopenjp2.so.7"
-    "/usr/local/lib/libzstd.so.1"
-    "/usr/local/lib/libz.so.1"
-)
-
-DEPS_SONAME=(
-    "libjpeg.so.62"
-    "libavformat.so.58"
-    "libavcodec.so.58"
-    "libavfilter.so.7"
-    "libavutil.so.56"
-    "libtiff.so.5"
-    "libsndfile.so.1"
-    "libFLAC.so.8"
-    "libogg.so.0"
-    "libvorbis.so.0"
-    "libvorbisenc.so.2"
-    "libopenjp2.so.7"
-    "libzstd.so.1"
-    "libz.so.1"
+    "/usr/${PATH_PREFIX}/lib64/libjpeg.so.62"
```

Comment:
```suggestion
    "${PATH_PREFIX}/lib64/libjpeg.so.62"
```

Reply:
For jetson it is `/usr/aarch64-linux-gnu/lib64/`. I can change the name to `LIB_PATH`.